tox.ini

```diff
@@ -36,7 +36,7 @@ deps =
     pytest-rerunfailures==9.0
 
 commands = pip install -i https://test.pypi.org/simple/ fetch-p2p-api==0.0.2
-           pytest -rfE --doctest-modules aea packages/fetchai/protocols packages/fetchai/connections tests/ --cov-report=html --cov-report=xml --cov-report=term --cov=aea --cov=packages/fetchai/protocols --cov=packages/fetchai/connections {posargs}
```

Comment:
We don't yet have full coverage for protocols in packages; so excluding them for now

Reply:
ok